# initial_configuration.yaml
motion_reference_generator:
  ros__parameters:

    initial_joint_position:
      # openarm_left
      openarm_left_joint1: -0.955
      openarm_left_joint2: -0.674
      openarm_left_joint3:  1.163
      openarm_left_joint4:  1.321
      openarm_left_joint5:  0.756
      openarm_left_joint6: -0.590
      openarm_left_joint7: -0.909
      # openarm_right: 
      openarm_right_joint1: 0.955
      openarm_right_joint2: 0.674
      openarm_right_joint3: -1.163
      openarm_right_joint4: 1.321
      openarm_right_joint5: -0.756
      openarm_right_joint6: 0.590
      openarm_right_joint7: 0.909

    initial_joint_velocity:
      # openarm_left
      openarm_left_joint1: 0.0
      openarm_left_joint2: 0.0
      openarm_left_joint3: 0.0
      openarm_left_joint4: 0.0
      openarm_left_joint5: 0.0
      openarm_left_joint6: 0.0
      openarm_left_joint7: 0.0
      # openarm_right
      openarm_right_joint1: 0.0
      openarm_right_joint2: 0.0
      openarm_right_joint3: 0.0
      openarm_right_joint4: 0.0
      openarm_right_joint5: 0.0
      openarm_right_joint6: 0.0
      openarm_right_joint7: 0.0

    initial_gripper_position:
      # openarm_left
      openarm_left_finger_joint1: 0.022 # max: 0.044
      # openarm_right
      openarm_right_finger_joint1: 0.022 # max: 0.044


robot_joint_dynamics:
  ros__parameters:

    initial_joint_position:
      # openarm_left
      openarm_left_joint1: -0.955
      openarm_left_joint2: -0.674
      openarm_left_joint3:  1.163
      openarm_left_joint4:  1.321
      openarm_left_joint5:  0.756
      openarm_left_joint6: -0.590
      openarm_left_joint7: -0.909
      # openarm_right: 
      openarm_right_joint1: 0.955
      openarm_right_joint2: 0.674
      openarm_right_joint3: -1.163
      openarm_right_joint4: 1.321
      openarm_right_joint5: -0.756
      openarm_right_joint6: 0.590
      openarm_right_joint7: 0.909

    initial_joint_velocity:
      # openarm_left
      openarm_left_joint1: 0.0
      openarm_left_joint2: 0.0
      openarm_left_joint3: 0.0
      openarm_left_joint4: 0.0
      openarm_left_joint5: 0.0
      openarm_left_joint6: 0.0
      openarm_left_joint7: 0.0
      # openarm_right
      openarm_right_joint1: 0.0
      openarm_right_joint2: 0.0
      openarm_right_joint3: 0.0
      openarm_right_joint4: 0.0
      openarm_right_joint5: 0.0
      openarm_right_joint6: 0.0
      openarm_right_joint7: 0.0